app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/StartController.scala

```diff
@@ -72,8 +76,24 @@ class StartController @Inject()(
                   case Some(knownFacts) => sessionStoreService.cacheKnownFactsResult(knownFacts)
                   case None             => Future.successful(())
                 }
+            (wasPartiallySubscribed, optAllocatedArn) <- knownFactsResultOpt match {
```

Comment:
There's a match on the `knownFactsResultOpt` on line 75. Could merge the match on line 79 into the match on line 75.

Reply:
done